common/src/main/java/com/wynntils/models/items/items/gui/ArchetypeItem.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.items.items.gui;
+
+import com.wynntils.models.items.properties.CountedItemProperty;
+import net.minecraft.ChatFormatting;
+
+public class ArchetypeItem extends GuiItem implements CountedItemProperty {
```

Comment:
I think this should be named `ArchetypeAbilitiesItem` (and correspondingly for the annotator), unless you plan to extend this item to contain more information in the future. (I don't remember from the top of my head if there are more information to be gained)